src/main/kotlin/inc/evil/d17map/controllers/ClassroomController.kt

```diff
@@ -94,6 +94,26 @@ class ClassroomController(private val classroomService: ClassroomService) {
         return ResponseEntity(updatedClassroom, HttpStatus.OK)
     }
 
+    @Operation(
+        summary = "Get photo of a classroom by ID",
+        responses = [
+            ApiResponse(responseCode = "200", description = "Successfully retrieved photo."),
+            ApiResponse(responseCode = "404", description = "Classroom with the given ID not found."),
+            ApiResponse(responseCode = "400", description = "Invalid classroom data provided.")
+        ]
+    )
+    @GetMapping("/{id}/photo")
+    fun getClassroomPhoto(@PathVariable id: UUID): ResponseEntity<ByteArray> {
+        val photo = classroomService.getClassroomPhotoById(id)
```

Comment:
Jakby w `getClassroomPhotoById` rzucić wyjątek na not Found, to mamy error handling, który się zajmie resztą 😁

Ifa wtedy można wrzucić I zwracać tylko pozytywne `ResponseEntity`